config/webpack.dev.config.js

```diff
@@ -16,12 +16,20 @@ targetUrl = `http://${targetUrl}:18010`;
 
 module.exports = Merge.smart(commonConfig, {
   devtool: 'cheap-module-eval-source-map',
-  entry: [
-    // enable react's custom hot dev client so we get errors reported
-    // in the browser
-    require.resolve('react-dev-utils/webpackHotDevClient'),
-    path.resolve(__dirname, '../src/index.jsx'),
-  ],
+  entry: {
+    assets: [
+      // enable react's custom hot dev client so we get errors reported
+      // in the browser
+      require.resolve('react-dev-utils/webpackHotDevClient'),
+      path.resolve(__dirname, '../src/index.jsx'),
+    ],
+    a11yPolicy: [
+      // enable react's custom hot dev client so we get errors reported
+      // in the browser
+      require.resolve('react-dev-utils/webpackHotDevClient'),
+      path.resolve(__dirname, '../src/a11yIndex.jsx'),
+    ],
+  },
   module: {
```

Comment:
For right now, if you want to test with the container, you will need to comment out the assets entry point so that the dev server will point to the accessibility index page instead

Reply:
*this* would actually be a good reason to implement routing via react-router or something similar. That way, we could keep a single entry point here and not have to worry about switching between pages by commenting things out.

Reply:
I think this is why I had initially defaulted to react-router in my thought process, before realizing the implications to integrating within the rest of studio/prod. I'd be happy to have a sit down (after this push) to create stories to handle our dev/prod in a cleaner way for future development as well. :)